src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        final Figure[] figures = new Figure[6];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < 3) {
+                figures[i] = figureSupplier.getRandomFigure();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+            }
+        }
+    }
```

Comment:
inside of this `for`, print the value that you generate, using `draw()` 
![image](https://github.com/mate-academy/jv-oop-advanced/assets/49767049/d362c35c-1cb2-4c1a-8ed5-1f0ed28ae9ac)